npc/instances/SealedShrine.txt

```diff
@@ -437,7 +437,7 @@ OnInstanceInit:
 	// Pick one "true" Gravestone and 12 "false" Gravestones.
 	.@true = rand(1,13);
 	for (.@i = 1; .@i<13; ++.@i) {
-		disablenpc instance_npcname("Gravestone#1F_1"+((.@i == .@true)?"F":"T"));
+		disablenpc instance_npcname("Gravestone#1F_"+.@i+((.@i == .@true)?"F":"T"));
```

Comment:
Found out other mistake,
should be
disablenpc instance_npcname("Gravestone#1F_"+.@i+((.@i == .@true)?"T":"F"));